component_layout_plugin.py

```diff
@@ -114,7 +114,7 @@ def Run( self ):
                 logger.warning("Did not find component {} in PCB design".format(refdes))
                 continue
             
-            flip = props.get('flip', False) # Generally, flip means put on the bottom
+            flip = props.get('flipped', False) # Generally, flip means put on the bottom
```

Comment:
Arguably 'flipped' is a better name for this property, but not sufficiently better to justify breaking backwards compatibility, so I don't see any reason to change this prop name...

Reply:
Oh I see I put 'flipped' in the README example, which is a mistake on my part.